config/rbac/role.yaml

```diff
@@ -214,6 +214,7 @@ rules:
   - serving.kserve.io
   resources:
   - inferencegraphs
+  - inferencegraphs/finalizers
```

Comment:
Also need a fix in the helm chart
https://github.com/kserve/kserve/blob/6921db56d1f1b0bc3c69ebe2028258f4d5aefae9/charts/kserve-resources/templates/clusterrole.yaml#L227